### DISCUSSION TITLE
Unable to access SiaCoin wallet

### DISCUSSION THREAD

**Discussion by captainkap** [siacentral/sia-lite-wallet-web#60] [Support] [answered]
Hello, I originally setup a custom password on my Sia Central Wallet and it has been working fine for about a year.  Now, all the sudden, when I login with my custom password, I get 4 options:

New Wallet
Recover Wallet
Ledger Wallet
Watch Only Wallet

I've tried the Recover and Ledger wallet options (I have a Ledger Nano S Plus), however, I don't know what the 29 word seed is being that I've always used a custom password for my wallet.

Are there any other options to recover my Sia wallet or is it gone forever?

**Comment by n8mgr** [comment, marked as answer]
If you used a Ledger, you don't need the 29-word recovery phrase. Instead, follow the instructions here https://docs.sia.tech/sia-integrations/using-the-sia-ledger-nano-app-sia-central to re-import your wallet.

If you have a standard wallet and did not save the 29-word recovery phrase, the wallet is lost. https://docs.sia.tech/your-sia-wallet/the-importance-of-your-seed

**Reply by captainkap** [reply]
I got it figured out using the Ledger instructions you provided, thanks!!
